<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RollWindow</class>
 <widget class="QWidget" name="RollWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="DigitalPlot" name="DI1" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI2" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI3" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI4" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI5" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI6" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI7" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI8" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI9" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI10" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI11" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI12" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI13" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI14" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI15" native="true"/>
     </item>
     <item>
      <widget class="DigitalPlot" name="DI16" native="true"/>
     </item>
     <item>
      <widget class="AnalogPlot" name="AI1" native="true"/>
     </item>
     <item>
      <widget class="AnalogPlot" name="AI2" native="true"/>
     </item>
     <item>
      <widget class="AnalogPlot" name="AI3" native="true"/>
     </item>
     <item>
      <widget class="AnalogPlot" name="AI4" native="true"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalogPlot</class>
   <extends>QWidget</extends>
   <header>analogPlot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DigitalPlot</class>
   <extends>QWidget</extends>
   <header>digitalPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
